shared/common-adapters/platform-icon.js.flow

```diff
@@ -8,6 +8,7 @@ export type Props = {
   size: 48,
   overlay: IconType,
   overlayColor: string,
+  style?: Object,
 }
 
 export default class Render extends Component<void, Props, void> { }
```

Comment:
You don't need this file, you can just put it in the normal platform-icon.js file